app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CommentsController < ApplicationController
+  def create
+    # ajout if logged in apres
+   Comment.create!(title:params[:name],content:params[:description],product_id:params[:product_id],user_id:current_user.id)
```

Comment:
Line is too long. [123/80]<br>Use 2 (not 1) spaces for indentation.<br>Space missing after colon.<br>Space missing after comma.